integration-tests/subscriptions.test.ts

```diff
@@ -86,6 +86,90 @@ if (SUBSCRIPTIONS_ENABLED === 'true') {
                 expect(notifications).toEqual([]);
             });
         }
+
+        test('end to end test with id notifications', async () => {
+            const uuid = v4();
+            const subResource = randomSubscription(uuid);
+            // 1. Create a Subscription.
+            const postSubscriptionResult = await client.post('Subscription', subResource);
+            expect(postSubscriptionResult.status).toBe(201);
+            // 2. Create/Update a resource that matches the subscription.
+            const postPatientResult = await client.post('Patient', resourceThatMatchesSubscription);
+            expect(postPatientResult.status).toEqual(201);
+            // 3. Verify that notifications are received
+            // give 2 minutes for notification to be placed in ddb table
```

Comment:
actually we need to wait one minute after creating the subscription but BEFORE posting the Patient that matches the subscription.(otherwise the Subscription may not be enabled yet due to caching)

I think that this needs to be fixed on tenant isolation tests as well